src/DataGrid.tsx

```diff
@@ -734,44 +746,6 @@ function DataGrid<R, SR, K extends Key>(
       setSelectedPosition({ ...position, mode: 'SELECT' });
     }
   }
-
-  function scrollToColumn(idx: number): void {
```

Comment:
This won't work with writing mode. Also, we do not need special handling for RTL if we use `scrollIntoView`